[11:53:53]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[12:43:08]Error in Signal_Main.py: name 'console_process' is not defined
[15:29:07]Error in console.py: name 'console_process' is not defined
[15:30:04]Error in console.py: name 'console_process' is not defined
[15:37:08]Error in console.py: module 'tkinter' has no attribute 'scrolledtext'
[15:37:16]Error in console.py: name 'scrolledtext' is not defined
[15:50:02]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[15:53:24]Error in Signal_Main.py: [Errno 2] No such file or directory: 'c:\\Users\\Tom�\\Desktop\\Tom�\\Dokumenty\\VSCode\\Python\\PyGame\\Signal_Bunker\\files\\functions\\temp.log'
[15:55:22]Error in Signal_Main.py: [Errno 2] No such file or directory: 'temp.log'
[16:00:21]Error in Signal_Main.py: [WinError 3] The system cannot find the path specified: 'logs/temp.log'
[16:00:56]Error in Signal_Main.py: [WinError 3] The system cannot find the path specified: 'logs\\temp.log'
[16:01:35]Error in Signal_Main.py: [WinError 3] The system cannot find the path specified: '\\logs\\temp.log'
[16:04:09]Error in Signal_Main.py: [WinError 3] The system cannot find the path specified: 'c:\\Users\\Tom�\\Desktop\\Tom�\\Dokumenty\\VSCode\\Python\\PyGame\\Signal_Bunker\\files\\files\\logs\\temp.log'
[16:10:31]Error in Signal_Main.py: [WinError 2] The system cannot find the file specified: 'C:\\Users\\Tom�\\Desktop\\Tom�\\Dokumenty\\VSCode\\Python\\PyGame\\Signal_Bunker\\files\\logs\\temp.log'
[08:50:44]Error in Signal_Main.py: [WinError 2] The system cannot find the file specified: 'C:\\Users\\Tom�\\Desktop\\Tom�\\Dokumenty\\VSCode\\Python\\PyGame\\Signal_Bunker\\files\\logs\\temp.log'
[09:11:04]Error in Signal_Main.py: name 'db' is not defined
[09:11:34]Error in Signal_Main.py: argument 1 must be pygame.surface.Surface, not str
[09:21:42]Error in Signal_Main.py: 'int' object is not subscriptable
[09:23:54]Error in Signal_Main.py: 'bool' object is not subscriptable
[20:38:54]Error in Signal_Main.py: name 'font' is not defined
[20:40:11]Error in Signal_Main.py: name 'text_surface' is not defined
[20:41:15]Error in Signal_Main.py: name 'text_surface' is not defined
[20:41:39]Error in Signal_Main.py: name 'text_surface' is not defined
[09:59:26]Error in Signal_Main.py: division by zero
[17:17:48]Error in Signal_Main.py: ord() takes exactly one argument (2 given)
[08:59:23]Error in Signal_Main.py: name 'tank_angle' is not defined
[08:19:44]Error in Signal_Main.py: name 'en_console' is not defined
[09:39:08]Error in Signal_Main.py: cannot access local variable 'command_frame' where it is not associated with a value
[09:45:39]Error in Signal_Main.py: name 'command' is not defined
[14:11:58]Error in console.py: [Errno 2] No such file or directory: 'c:\\Users\\Tom�\\Desktop\\Tom�\\Dokumenty\\VSCode\\Python\\PyGame\\Signal_Bunker\\files\\functions\\..\\logs\\temp.log'
[14:12:41]Error in console.py: [Errno 2] No such file or directory: 'c:\\Users\\Tom�\\Desktop\\Tom�\\Dokumenty\\VSCode\\Python\\PyGame\\Signal_Bunker\\files\\functions\\logs\\temp.log'
[14:15:39]Error in console.py: name 'command_line' is not defined
[15:24:20]Error in Signal_Main.py: module 'functions.data' has no attribute 'tank_angle'
[15:27:39]Error in Signal_Main.py: module 'functions.data' has no attribute 'tank_angle'
[15:30:41]Error in Signal_Main.py: '>' not supported between instances of 'str' and 'int'
[15:32:37]Error in Signal_Main.py: '>' not supported between instances of 'str' and 'int'
[15:43:59]Error in Signal_Main.py: module 'functions.data' has no attribute 'tank_angle'
[15:46:37]Error in Signal_Main.py: module 'functions.data' has no attribute 'tank_angle'
[08:44:48]Error in Signal_Main.py: division by zero
[08:49:56]Error in Data.py: Error during file loading: No such file or directory: 'c:\Projekts\School_Project\files\functions\..\textures\Surface.jp'.
[08:49:59]Error in Signal_Main.py: module 'functions.data' has no attribute 'surface'
[16:40:58]Error in Signal_Main.py: cannot access local variable 'gen_y' where it is not associated with a value
[16:41:59]Error in Main.py: cannot access local variable 'gen_y' where it is not associated with a value
[16:42:59]Error in Main.py: cannot access local variable 'gen_x' where it is not associated with a value
[16:44:28]Error in Main.py: cannot access local variable 'gen_y' where it is not associated with a value
[16:47:24]Error in Main.py: cannot access local variable 'gen_y' where it is not associated with a value
[16:50:25]Error in Main.py: cannot access local variable 'gen_x' where it is not associated with a value
[16:50:51]Error in Main.py: cannot access local variable 'gen_y' where it is not associated with a value
[17:00:14]Error in Main.py: generation() takes 1 positional argument but 3 were given
[08:20:39]Error in Main.py: [WinError 2] The system cannot find the file specified: 'c:\\Projekts\\School_Project\\files\\logs\\temp.log'
[08:20:52]Error in Main.py: [WinError 2] The system cannot find the file specified: 'c:\\Projekts\\School_Project\\files\\logs\\temp.log'
[07:45:37]Error in Data.py: Error during file loading: mixer not initialized
[07:46:32]Error in Data.py: Error during file loading: Cannot scale to negative size
[07:47:10]Error in Main.py: module 'functions.data' has no attribute 'red_shell'
[07:47:35]Error in Data.py: Error during file loading: Cannot scale to negative size
[07:47:37]Error in Main.py: module 'functions.data' has no attribute 'red_shell'
[07:48:07]Error in Data.py: Error during file loading: mixer not initialized
[07:48:18]Error in Data.py: Error during file loading: mixer not initialized
[07:48:29]Error in Data.py: Error during file loading: mixer not initialized
[07:48:52]Error in Data.py: Error during file loading: mixer not initialized
[07:50:36]Error in Data.py: Error during file loading: No such file or directory: 'c:\Projekts\School_Project\files\functions\..\textures\cannon_shot.wav'.
[08:12:59]Error in Data.py: Error during file loading: invalid syntax (<string>, line 7)
[08:13:10]Error in Main.py: module 'functions.data' has no attribute 'surface'
[07:48:46]Error in Data.py: Error during file loading: No such file or directory: 'c:\Projekts\School_Project\files\functions\..\sounds\clam_engine.wav'.
[07:48:47]Error in Main.py: module 'functions.data' has no attribute 'calm_engine'
[08:15:51]Error in Main.py: 'NoneType' object has no attribute 'play'
[08:16:28]Error in Main.py: 'NoneType' object has no attribute 'play'
[10:59:11]Error in db.py: connection to server at "192.168.43.10", port 5432 failed: Connection timed out (0x0000274C/10060)
[09:38:44]Error in Main.py: expected str, bytes or os.PathLike object, not NoneType
[09:39:22]Error in db.py: connection to server at "0.0.0.0", port 5432 failed: Cannot assign requested address (0x00002741/10049)
	Is the server running on that host and accepting TCP/IP connections?
[09:42:17]Error in Main.py: Unknown format code 'd' for object of type 'float'
[10:05:31]Error in db.py: connection to server at "0.0.0.0", port 5432 failed: Cannot assign requested address (0x00002741/10049)
	Is the server running on that host and accepting TCP/IP connections?
[10:13:04]Error in Main.py: Unknown format code 'd' for object of type 'float'[12:13:28]Error in Main.py: 'bool' object has no attribute 'cursor'
[12:14:44]Error in Main.py: 'bool' object has no attribute 'cursor'
[12:16:28]Error in Main.py: 'bool' object has no attribute 'cursor'
[12:17:32]Error in Main.py: 'bool' object has no attribute 'cursor'
[12:27:11]Error in Main.py: clound_load() takes 0 positional arguments but 1 was given
[13:45:47]Error in Main.py: [Errno 2] No such file or directory: 'files\\saves\\save_0.sav'
[13:47:20]Error in Main.py: list index out of range
[13:50:07]Error in Main.py: write() argument must be str, not list
[13:56:16]Error in Main.py: invalid literal for int() with base 10: ''
[13:59:43]Error in Main.py: invalid literal for int() with base 10: ''
[14:01:30]Error in Main.py: invalid literal for int() with base 10: ''
[14:04:11]Error in Main.py: invalid literal for int() with base 10: ''
[14:05:06]Error in Main.py: invalid literal for int() with base 10: ''
[14:06:19]Error in Main.py: invalid literal for int() with base 10: ''
[15:39:21]Error in Main.py: invalid literal for int() with base 10: ''
[15:43:05]Error in Main.py: invalid literal for int() with base 10: ''
[15:47:04]Error in save.py: 'bool' object is not subscriptable
[15:47:58]Error in save.py: 'bool' object is not subscriptable
[15:52:39]Error in save.py: '<' not supported between instances of 'str' and 'int'
[15:55:59]Error in save.py: 'bool' object is not subscriptable
[15:57:07]Error in save.py: 'bool' object is not subscriptable
[15:59:54]Error in save.py: 'bool' object is not subscriptable
[16:10:56]Error in save.py: 'bool' object is not subscriptable
[16:14:29]Error in save.py: 'bool' object is not subscriptable
[16:15:15]Error in save.py: 'bool' object is not subscriptable
[16:15:54]Error in save.py: 'bool' object is not subscriptable
[16:16:25]Error in save.py: 'bool' object is not subscriptable
[16:25:59]Error in save.py: 'bool' object is not subscriptable
[16:38:23]Error in save.py: 'float' object has no attribute 'strip'
[16:43:35]Error in save.py: 'float' object has no attribute 'strip'
[14:03:41]Error in Main.py: 0
[14:05:09]Error in Main.py: 0
[08:49:10]Error in save.py: Save file is corrupted or incomplete.
[18:19:38]Error in account_logic.py: Username already exists
[18:20:56]Error in Main.py: No file 'fonts\impact.ttf' found in working directory 'C:\Projekts\School_Project'.
